coralproject/talk
# [CORL-1527] Default to pending queue if global premod is enabled

## What does this PR do?

If `settings.moderation` is set to `PRE`: 
- redirect /admin/moderate to /admin/moderate/pending instead of /reported
- hide reported queue



## What changes to the GraphQL/Database Schema does this PR introduce?
none


## How do I test this PR?

- toggle `settings.moderation` 
- if `PRE`: click "moderate" in navigation, should go to pending queue and hide reported queue
- if `POST`: click "moderate" in navigation, should go to reported queue
